frontend/src/services/mock.ts

```diff
@@ -24,4 +28,35 @@ export const mockUser: User = {
 export const mockWallet: IWallet = {
     id: 40,
     balance: 100,
+}
+
+export const mockExpensesTransactions: IExpenseTransaction = {
+    transactions: [
+        {
+            amount: 30,
+            nameTo: "Starbucks",
+        },
+        {
+            amount: 20,
+            nameTo: "McDonald's",
+        },
+        {
+            amount: 20,
+            nameTo: "Steam",
+        },
+        {
+            amount: 70,
+            nameTo: "Others"
+        },
+    ],
+    amount: 140
+}
+
+export const mockIncome: IIncome = {
+    amount: 30
+}
+
+export const mockTransactionsData: ITransactionData = {
+    expensesTransactions: mockExpensesTransactions,
```

Comment:
expencesTransactions.transactions duplication.
Also below you have income, it means here should be expences
```suggestion
    expenses: mockExpensesTransactions,
```